sos/plugins/openstack_cinder.py

```diff
@@ -27,8 +27,10 @@ class OpenStackCinder(Plugin):
     plugin_name = "openstack_cinder"
     profiles = ('openstack', 'openstack_controller')
 
-    option_list = [("db", "gathers openstack cinder db version", "slow",
-                    False)]
+    option_list = [
+        ("db", "gathers openstack cinder db version", "slow", False),
+        ("exclude", "Ignore files matching this path spec", "", "")
+    ]
```

Comment:
Thanks for your feedback Bryn.

Let me  give some additional background here around the file namings.

I think it's reasonable to assume each service's logs are in /var/log/<openstack-service>. (This is what the existing plugins do).

The tricky bit is the file names. OpenStack leaves it up to the user to decide what the machine readable file names will be (or if you want to generate them at all). So HPE customers will end up with glance-api-json.log for example, but I don't know what the equivalent Red Hat ones would be called, whether they'd have json in the name or if Red Hat's OpenStack distro even generates json logs at all.

(As an aside: the regular (non-json) file names are also not known, but that hasn't been a problem because sosreport has just been collecting the full directory.)

I appreciate that this seems to jar a little with the way sosreport typically works...

It sounds like what you're suggesting would work ... as long as the option to remove json log files could work even when json isn't in the file names.

But in that case, I guess it could be used to remove any files, and I'm not sure how that fits with sos's conventions. (Also, if that were the case there'd be no reason to say it's a 'json' specific option.)

It  may be the case that we just end up carrying a downstream patch for this (not the end of the world.)

Thanks again!